src/solver/infeasible-problem-analysis/watched-constraints.cpp

```diff
@@ -159,17 +164,22 @@ std::string HydroProduction::infeasibilityCause()
 ConstraintsFactory::ConstraintsFactory()
 {
     regex_to_ctypes_ = {
-      {"::hourly::", std::make_unique<HourlyBC, const std::string&, const double>},
-      {"::daily::", std::make_unique<DailyBC, const std::string&, const double>},
-      {"::weekly::", std::make_unique<WeeklyBC, const std::string&, const double>},
-      {"^FictiveLoads::", std::make_unique<FictitiousLoad, const std::string&, const double>},
-      {"^AreaHydroLevel::", std::make_unique<HydroLevel, const std::string&, const double>},
-      {"^Level::", std::make_unique<STS, const std::string&, const double>},
-      {"^HydroPower::", std::make_unique<HydroProduction, const std::string&, const double>}};
+      {"::hourly::", std::make_unique<HourlyBC, const std::string&, double>},
+      {"::daily::", std::make_unique<DailyBC, const std::string&, double>},
+      {"::weekly::", std::make_unique<WeeklyBC, const std::string&, double>},
+      {"^FictiveLoads::", std::make_unique<FictitiousLoad, const std::string&, double>},
+      {"^AreaHydroLevel::", std::make_unique<HydroLevel, const std::string&, double>},
+      {"^Level::", std::make_unique<STS, const std::string&, double>},
+      {"^HydroPower::", std::make_unique<HydroProduction, const std::string&, double>},
+      {"^WithdrawalSum::", std::make_unique<STSWithdrawalSum, const std::string&, double>},
+      {"^InjectionSum::", std::make_unique<STSInjectionSum, const std::string&, double>},
+      {"^NettingSum::",
+       std::make_unique<STSNettingSum, const std::string&, double>}}; // namespace
+                                                                      // Antares::Optimization
```

Comment:
We should remove comments at end of line 177. It seems useless and causes clang to add a new line.

Reply:
done (not on purpose)